src/betterproto/__init__.py

```diff
@@ -664,7 +666,8 @@ def _cls_for(cls, field: dataclasses.Field, index: int = 0) -> Type:
         """Get the message class for a field from the type hints."""
         field_cls = cls._type_hint(field.name)
         if hasattr(field_cls, "__args__") and index >= 0:
-            field_cls = field_cls.__args__[index]
+            if field_cls.__args__ is not None:
```

Comment:
Is there not something better we can do here? Won't this mean we just miss the annotation on 3.6?

Reply:
Don't think so, the difference is that `__args__` does not exist in 3.7+ and in the same scenario it returns `None` in 3.6. Not sure who behavior has changed here, we are just being stricter with the condition to avoid a `ValueError`.